lp-core-platform/lp-cp-apis/src/main/java/eu/learnpad/core/impl/qm/XwikiBridgeInterface.java

```diff
@@ -29,24 +29,24 @@
 public class XwikiBridgeInterface implements XWikiRestComponent, BridgeInterface{
 
 	@Override
-	public void putModel(String modelId, String type, byte[] bpmnFile)
-			throws LpRestExceptionImpl {
+	public void importModelSet(String modelSetId, String type,
+			byte[] modelContent) throws LpRestExceptionImpl {
 		// TODO Auto-generated method stub
 		
 	}
 
 	@Override
-	public byte[] getQuestionnaires(String modelId, String type, byte[] bpmnFile)
-			throws LpRestExceptionImpl {
+	public String generateQuestionnaires(String modelSetId, String type,
+			byte[] configurationFile) throws LpRestExceptionImpl {
```

Comment:
The configuration may be sent as a Java class instead of as a file.  However, we still don't know what will be this class.  You may just want to add a comment for future expectations.